Description: somehow pybuild doesnt build this properly as a shared lib

--- pybdsf-1.8.14.orig/setup.py
+++ pybdsf-1.8.14/setup.py
@@ -103,7 +103,8 @@ def main():
     fext=Extension(
         name="bdsf._pytesselate",
         sources=["src/fortran/pytess_simple.f",
-                 "src/fortran/pytess_roundness.f"]
+                 "src/fortran/pytess_roundness.f"],
+        extra_link_args=['-shared'],
         );
     fext.f2py_options=[""]
     extensions.append(fext)
